common/src/main/java/earth/terrarium/hermes/utils/RenderUtils.java

```diff
@@ -0,0 +1,21 @@
+package earth.terrarium.hermes.utils;
+
+import net.minecraft.client.gui.GuiGraphics;
+
+public final class RenderUtils {
+
+    public static void renderOutline(GuiGraphics graphics, int x, int y, int outerWidth, int outerHeight, int color, int thickness) {
```

Comment:
These should just be in the following order and name to match consistency 
- x
- y
- width
- height
- thickness
- color

Variables should also be on their own lines not all on a single line.
And personally this is just my opinion but the variables should be x0 x1 y0 y1 as to follow correct UV naming schemes.

Reply:
Will change. For me education: consistency to what?
I was just riffing off GuiGraphics.renderOutline()'s x, y, i, j, placing the thickness last because that's what our renderOutline adds.

Though I would gently argue for "outerWidth" as being more descriptive/self-documenting.

p.s. I think I already collapsed ("resolved") the sub-thread here about changing that anonymous function to a method, but... I just noticed a superfluous semi-colon there; it's valid syntax but will remove it in the next commit. Currently working on the parsing stuff for FillAndBorder.

Reply:
For consistency if you look at all the rendering methods in GuiGraphics all of them try to stick size, position and general constraints like that similar to each other next to each for parameters.

Personally outerWidth imo doesnt make too much sense, outer of what? width and height is what Parchment calls it and makes the most sense I dont think adding outer the method is just the same as renderOutline but has a thickness argument.

Reply:
Huh. This is way I'm thinking of it: an outline with a thickness, in plain language, has both an inner and outer width and height. Imagine a real life painting frame: the inner width is the negative space where the painting would show.

Regardless, it's subjective and dependant on what mental models one is already bringing to the situation—and I'm just contributing/a guest, so happy to go with your take.

Commit for using plain "width" and "height" forthcoming.

Reply:
9adf2fa has it. ^^